ios/FluentUI.Demo/FluentUI.Demo/Demos/TabBarViewDemoController.swift

```diff
@@ -74,12 +74,14 @@ class TabBarViewDemoController: DemoController {
         updatedTabBarView.delegate = self
 
         if showsItemTitles {
+            homeItem = TabBarItem(title: "Home", image: UIImage(named: "Home_24")!, selectedImage: UIImage(named: "Home_Selected_24")!)
```

Comment:
The initialization of the lazy var homeItem is a bit different than these 2 calls being introduced:

```
    private lazy var homeItem: TabBarItem = {
        return TabBarItem(title: "Home", 
                          image: UIImage(named: "Home_28")!, 
                          selectedImage: UIImage(named: "Home_Selected_28")!,
                          landscapeImage: UIImage(named: "Home_24")!,
                          landscapeSelectedImage: UIImage(named: "Home_Selected_24")!)
    }()
```

Have you tested it in landscape mode?

Reply:
yes I have tested and I did this intentionally because now it is correct and earlier it was wrong because when we show titles then we should be using _24 asset otherwise _28 asset and in landscape it should always be _24 asset.